### PR TITLE
[CI:DOCS] Bump to v2.2.0-rc2

### PR DESCRIPTION
**Pull request by rhatdan**

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,27 @@
+- Changelog for v2.2.0-rc2 (2020-11-23):
```
Comment (mheon) [inline changelog.txt:1]: No - this bit goes in RELEASE_NOTES.md - changelog is made with `make changelog`

Comment (mheon) [reply changelog.txt:1]: Honestly you could just leave the release notes off until the final - wouldn't the the first time.

**Comment by rhatdan** [comment]
@containers/podman-maintainers PTAL
@vrothberg @saschagrunert @giuseppe PTAL

**Review by vrothberg — COMMENTED** [review]
LGTM

**Pull request merged**